ansible/es-7.yml

```diff
@@ -0,0 +1,4 @@
+---
+- hosts: es-7
```

Comment:
change the host to es7

Reply:
Raj i have made the changes, thanks

Reply:
but the host is still es7.

Reply:
now you check rajesh it is updated, two es7 files is used be there, i have removed that one